src/adapter/src/catalog/builtin.rs

```diff
@@ -1269,7 +1269,15 @@ UNION
 UNION
     SELECT mz_indexes.id, mz_indexes.oid, schema_id, mz_indexes.name, 'index'
     FROM mz_catalog.mz_indexes
-    JOIN mz_catalog.mz_relations ON mz_indexes.on_id = mz_relations.id",
+    JOIN mz_catalog.mz_relations ON mz_indexes.on_id = mz_relations.id
+UNION
```

Comment:
I'm on board with cargo culting here but also: I think the various `UNION`s in all of these views could (should?) be `UNION ALL` which allows compute to produce a better plan (doesn't have to distinct rows). Unverified assumption: there aren't any duplicate rows here, which seems correct?

Fine leaving as is or doing in another PR that changes all of these (for example I see mz_relations above, probably others).

Reply:
I was going to make the same suggestion until I saw that all other views used `UNION` as well, and assumed there was some valid reason for that; but agree with Matt on using `UNION ALL` if not.

Reply:
Thanks for the tip! I can make the change here - it doesn't seem like there are that many instances that need to be fixed.